src/Orc.NuGetExplorer.Xaml/ViewModels/PackageDetailsViewModel.cs

```diff
@@ -238,29 +271,43 @@ protected override Task CloseAsync()
             return base.CloseAsync();
         }
 
-        private void OnNuGetActionTargetPropertyPropertyChanged(object sender, System.ComponentModel.PropertyChangedEventArgs e)
+        private void OnNuGetActionTargetPropertyPropertyChanged(object? sender, PropertyChangedEventArgs e)
         {
             var commandManager = this.GetViewModelCommandManager();
             commandManager.InvalidateCommands();
         }
 
-        protected async override void OnPropertyChanged(AdvancedPropertyChangedEventArgs e)
+        protected async override void OnPropertyChanged(PropertyChangedEventArgs e)
         {
             base.OnPropertyChanged(e);
 
-            if (string.Equals(e.PropertyName, nameof(SelectedVersion)))
+            if (e.HasPropertyChanged(nameof(SelectedVersion)))
             {
-                if ((e.OldValue is null && SelectedVersion == Package.Identity.Version) || e.NewValue is null)
+                if (e is PropertyChangedExtendedEventArgs<NuGetVersion> args)
+                {
+                    if ((args.OldValue is null && SelectedVersion == Package?.Identity.Version) || args.NewValue is null)
```

Comment:
this seems new code. Make sure to cover this with unit tests to ensure correct behavior.